### PR TITLE
Mailing-list-landing-pages-variants-for-split-testing

### PR DESCRIPTION
**Pull request by MaxineCl**
### Trello card
https://trello.com/c/lDSHurFE/5717-create-variants-for-mailing-list-landing-page-split-test 

### Context
The split tests we have run for a variant mailing list landing page have not shown positive results so we want to test some alternate variants.

### Changes proposed in this pull request
We want to create three further variants to test where we:

- shorten body copy
- remove image and bullets
- different CTA text

### CODE REVIEW

**Comment by MaxineCl** [comment]
@jenhadfield-dfe would you mind taking a look at variant 2 and deleting the image and H1 as I can't work out how to do it. Thanks

**Comment by MaxineCl** [comment]
@gemmadallmandfe just wondering on version 1 if we lose this bit too? 

Whether you’re just thinking about teaching or you’re ready to apply, we’re here to guide you through the steps to becoming a teacher.

So we just have the H2 and bullets?

File: app/views/content/landing/mailing-list-1/_mailing-list.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <section class="col col-720 col-space-0">
+      <%= render Content::MailingListComponent.new(
+        title: "Get free advice and support tailored to you",
+        color: "transparent",
+        margin: false,
+        heading: :l
+      ) do %>
+
+        <p>
+          Whether you’re just thinking about teaching or you’re ready to apply, we’re here to guide you through the steps to becoming a teacher.
```
Comment (gemmadallmandfe) [inline app/views/content/landing/mailing-list-1/_mailing-list.html.erb:12]: @MaxineCl Agree with your suggestion about losing this paragraph on variant 1

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**